src/app/payment/storedInstrument/ManageInstrumentsModal.tsx

```diff
@@ -86,7 +86,7 @@ class ManageInstrumentsModal extends Component<ManageInstrumentsModalProps & Wit
         const accountInstruments = instruments.filter(isAccountInstrument);
 
         return (
-            accountInstruments.length
+            accountInstruments.length || bankInstruments.length
```

Comment:
Don't we need to add a unit test to _ManageInstrumentsModal.spec.tsx_ to check that `ManageAccountInstrumentsTable` is rendered when `filter(isBankAccountInstrument)` returns some records?

Reply:
You're right, unit test was added.